docs/advanced-topics/serialization/inetworkserializebymemcpy.md

```diff
@@ -0,0 +1,29 @@
+---
+id: inetworkserializebymemcpy
+title: INetworkSerializeByMemcpy
+sidebar_label: INetworkSerializeByMemcpy
+---
+
+The `INetworkSerializeByMemcpy` interface is used as a sort of "tag" to mark an unmanaged struct type as being serializable over the network by directly copying the whole struct, byte-for-byte, as it appears in memory, into and out of the buffer. This can offer some benefits for performance compared to serializing one field at a time, especially if the struct has many fields in it.
```

Comment:
> The `INetworkSerializeByMemcpy` interface is used as a sort of "tag" to mark an unmanaged struct type...
Maybe?
The `INetworkSerializeByMemcpy` interface is used to mark an unmanaged struct type...

Reply:
I would say not quite. Saying it "marks the type as serializable over the network" would make me think I would need to use that for anything I plan to serialize - but that's not true, as `INetworkSerializable` is also an option, as is custom serialization. It marks a type for a _particular type_ of serialization.